packages/procaptcha-bundle/src/index.tsx

```diff
@@ -195,12 +195,12 @@ const implicitRender = () => {
 }
 
 // Explicit render for targeting specific elements
-export const render = (elementId: string, renderOptions: ProcaptchaRenderOptions) => {
+export const render = (element: Element, renderOptions: ProcaptchaRenderOptions) => {
     const siteKey = renderOptions.siteKey
-    const element = document.getElementById(elementId)
+    // const element = document.getElementById(elementId)
 
     if (!element) {
-        console.error('Element not found:', elementId)
+        console.error('Element not found:', element)
```

Comment:
I'm not sure that this code path can be reached - the element is passed to render, which I believe guarantees that it exists

Reply:
Okay, I will remove